bin/setup_scale

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash -e
+
+ROOT=$(dirname $0)/..
+cd $ROOT
+local=
+devices=
+switches=
+skip_conn_check=
```

Comment:
This script likely has a lot of stuff in it that's no longer relevant?  E.g. is skip_conn_checked needed?  Can it be the default? etc...  Is it possible to strip this down to just the salient bits?  It's easier to add things back in down the road if necessary, but it's typically harder to remove them later (since then you don't remember what was important now).

Reply:
Got it

Reply:
I stripped off skip-conn-check but I think other elements would probably be needed as we go. Local for local tests, fot, etc would almost certainly play a role in further scaling tests.

Reply:
sgtm -- as long as we tried!